libs/pilight/protocols/433.92/iwds07.c

```diff
@@ -0,0 +1,120 @@
+/*
+	Copyright (C) 2017 CurlyMo & wo-rasp
+
+	This file is part of pilight.
+
+	pilight is free software: you can redistribute it and/or modify it under the
+	terms of the GNU General Public License as published by the Free Software
+	Foundation, either version 3 of the License, or (at your option) any later
+	version.
+
+	pilight is distributed in the hope that it will be useful, but WITHOUT ANY
+	WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+	A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with pilight. If not, see	<http://www.gnu.org/licenses/>
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+#include "../../core/pilight.h"
+#include "../../core/common.h"
+#include "../../core/dso.h"
+#include "../../core/log.h"
+#include "../../core/binary.h"
+#include "../../core/gc.h"
+#include "../protocol.h"
+#include "iwds07.h"
+
+#define PULSE_MULTIPLIER        3
+#define MIN_PULSE_LENGTH        415
+#define MAX_PULSE_LENGTH        1245
+#define AVG_PULSE_LENGTH        830
+#define	FOOTER			14110
```

Comment:
>What is the recommended spacing for TAB setting ?

That's the point of tabs 😉
everyone can set them to the width they prefer, but it will always be consistent throughout files. This setting only matters if your editor wants to mess with the indentation throughout the entire file, which it shouldn't.